tiup/tiup-reference.md

```diff
@@ -0,0 +1,87 @@
+---
+title: TiUP Reference
+---
+
+# TiUP Reference
+
+TiUP serves as the package manager of the TiDB ecosystem. It manages components in the TiDB ecosystem, such as TiDB, PD, and TiKV.
+
+## Syntax
+
+```shell
+tiup [flags] <command> [args...]        # Executes a command
+# or
+tiup [flags] <component> [args...]      # Runs a component
+```
+
+You can use the `help` command to get the information of a specific command. The summary of each command shows its parameters and their usage. Mandatory parameters are shown in angle brackets, and optional parameters are shown in square brackets.
+
+`<command>` represents the command name. For the list of supported commands, see the [Command list](#command-list) below. `<component>` represents the component name. For the list of supported components, see the [Component list](#component-list) below.
+
+## Options
+
+### -B, --binary
+
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command, and pass the `true` value or do not pass any value.
+- If you enable this option, the specified binary file path is printed.
+
+    - Executing `tiup -B/--binary <component>` will have the path of the latest stable installed `<component>` component printed. If `<component>` is not installed, an error is returned.
+    - Executing `tiup -B/--binary <component>:<version>` will have the path of the installed `<component>` component's `<version>` printed. If this `<version>` is not printed, an error is returned.
+
+> **Note:**
+>
+> This option can only be used in commands of the `tiup [flags] <component> [args...]` format.
+
+### --binpath
+
+- Data type: `STRING`
+- Specifies the path of the component to be executed. When a component is executed, if you do not want to use the binary file in the TiUP mirror, you can add this option to specify using the binary file in a custom path.
+
+> **Note:**
+>
+> This option can only be used in commands of the `tiup [flags] <component> [args...]` format.
+
+### --skip-version-check
+
+> **Note:**
+>
+> This option is deprecated since v1.3.0.
+
+- Data type: `BOOLEAN`
+- This option is disabled by default and its default value is `false`. To enable this option, you can add this option to the command, and pass the `true` value or do not pass any value.
+- Skips the validity check for version numbers. By default, the specified version number can only be the semantic version.
+
+### -T, --tag
+
+- Data type: `STRING`
+- Specifies a tag for the component to be started. Some components need to use disk storage during the execution, and TiUP will allocate a temporary storage directory for this execution. If you want TiUP to allocate a fixed directory, you can use `-T/--tag` to specify the name of the directory, so that the same batch of files can be read and written in multiple executions with the same tag.
```

Comment:
```suggestion
- Specifies a tag for the component to be started. Some components need to use disk storage during the execution, and TiUP allocates a temporary storage directory for this execution. If you want TiUP to allocate a fixed directory, you can use `-T/--tag` to specify the name of the directory, so that the same batch of files can be read and written in multiple executions with the same tag.
```